src/osd/OSD.cc

```diff
@@ -5834,8 +5834,40 @@ bool OSD::scrub_random_backoff()
   return false;
 }
 
+bool OSD::scrub_time_permit()
```

Comment:
I was thinking of testing both osd_scrub_begin_hour < and > osd_scrub_end_hour.  So you would have I would test the following:

current time  osd_scrub_begin_hour    osd_scrub_end_hour    result
01:00